blog/_posts/Team-Hackrgirls.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing Team Hackrgirls
+layout: post
+created_at: Saturday August 1st 2015
+permalink: blog/introducing-Team-Hackrgirls
+current: blog
+author: Nidhiya and Anjaly
+twitter: _tyranja_
+---
+
+
+<a href="https://nidhiya.files.wordpress.com/2015/07/665572570.jpg"><img class="alignleft size-full wp-image-277" src="https://nidhiya.files.wordpress.com/2015/07/665572570.jpg" alt="665572570" width="826" height="709" /></a>
```

Comment:
could you add this image to the `img/blog/2015` folder as described in the [readme](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/README.md), and then change the link accordingly? Would be great, thank you! :)